third_party/pyth/prepare_multisig.py

```diff
@@ -0,0 +1,19 @@
+# This script prepares a local Squads multisig deployment for use with
```

Comment:
Do we really need to have this? 

We can just append `nc -lkp <port> 0.0.0.0` to the pod command to have readiness.

Reply:
I think it would be nice to be able to orchestrate some scenarios on the multisig - e.g. add some new members and start governance actions in the vault and have mock multisig members approve them. This is different from the `p2w_autoattest` script which is used in prod (which I want to make go away eventually).

Reply:
Got it. make sense then. 👍